AppController.m

```diff
@@ -771,9 +771,13 @@ -(BOOL)control:(NSControl *)control textView:(NSTextView *)fieldEditor doCommand
 
 -(BOOL)shouldSkip:(NSString *)contents
 {
-	NSString *type = [jcPasteboard availableTypeFromArray:[NSArray arrayWithObject:NSStringPboardType]];
+    // main type
+    NSString *type = [jcPasteboard availableTypeFromArray:[NSArray arrayWithObject:NSStringPboardType]];
```

Comment:
I'm not sure we even need to keep this type variable. The only place it is used after this change is in a optional log, which should probably print out ` [pboardTypes componentsJoinedByString:@","]` anyway. 

Since the useful type to ignore will most likely not be the first one in the list. 
Line 785: `[clippingStore addClipping:type ofType:type fromAppLocalizedName:@"Flycut" fromAppBundleURL:nil atTimestamp:0];`